### PR TITLE
management week and goals for week 3

### PR DESCRIPTION
**Pull request by keyonzi**

### CODE REVIEW

File: management/Roguelike.md [hunk 1]
```diff
@@ -11,35 +11,37 @@
 3. 06/16 - 06/22
 	* Manager: NAME
```
Comment (nfarnan) [inline management/Roguelike.md:12]: Who is the manger for week 3? They need to be the one to issue the PR with their goals.

File: management/Roguelike.md [hunk 2]
```diff
@@ -11,35 +11,37 @@
...
+		1. Outline for procedural generation preso
 		...
 4. 06/23 - 06/29
 	* Manager: NAME
 	* Goals:
 		1. GOAL1
 		...
 5. 06/30 - 07/6
-	* Manager: NAME
+	* Manager: Trae Yelovich
```
Comment (nfarnan) [inline management/Roguelike.md:24]: Please add Pitt IDs (e.g., nlf4) to each manager for easy lookup on my end (should be in the team roster description).

**Review by nfarnan — CHANGES_REQUESTED** [review]
Some of the changes that I'm requesting will require undoing old commits. You're going to have to checkout a previous commit and start a new history from there. Avoiding this difficulty is one reason to work with branches for any new changes. As an easy short cut, you may just want to re-clone the repo to start fresh, and work on branches from there.

Also, you should not submit goals for management weeks that are not your own, and while I will accept you filling out the entire management week schedule, the expectation was that each student would submit a PR for their own week (i.e., if you don't want to look up everyone's Pitt IDs, tell them all to do it themselves and send me individual PRs)

**Comment by keyonzi** [comment]
Hi Professor,

You recommended for our canonical repo to also do forks and not branches. Which is what I told them. They went ahead with the organization, but as I understood from you, we should fork and not branch. So just a bit confused on that. 

Keyon

**Comment by nfarnan** [comment]
These techniques are not mutually exclusive, you can have both a fork and branches. I would discourage teams from using only branches on the canonical repository for organization, but I would still encourage you to create branches on your for as you develop new features. This chapter from the git book should give a good description of how you can use branches effectively even as the sole developer of a project:

https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging

**Comment by keyonzi** [comment]
Appreciate it. 

Btw, I withdrew from the other class. Hope it was the smart decision. 

Thanks,
Keyon

**Comment by nfarnan** [comment]
OK, we still have a few issues here.

1. This is the organization repo, it does not need the .gitignore
2. You still have changes to the lines for other management weeks (e.g, from "NAME" to ""). I realize this sounds overly particular, but this means that any other pull requests on these lines would need to either a) rebase off of your changes before issuing the PR or b) I have to resolve a merge conflict on those lines. I realize this unnecessarily difficult to just sign up for management weeks, but I would rather have you guys make errors like this in the organization repo than to have them come up a couple of weeks later in your game repo. You can try either resetting your working branch and retrying, or just delete and re-clone the org repo to issue another pull request. This one has gotten a bit convoluted, so I'm closing it.

**Comment by keyonzi** [comment]
the gitignore was an accident. I was trying to add it to our roguerepo and
accidently added to the wrong place. I thought I had removed it.

I will try and fix the 'NAME' issue

**Pull request closed without merging**